### ISSUE TITLE
Panic at login in Fedora

### ISSUE THREAD

**Issue by eaceto** [cloudfoundry/cli#848]
I'm running cf in Fedora 24 (golang 1.6.2) and I have the following panic when trying to login

### Command
[kimi@ThinkCentre gcba-bot]$  cf login -u <My User>

### Response
API endpoint: https://api.ng.bluemix.net

### CLI Version
cf version 6.18.1+a1103f0-2016-05-24

### Stack Trace
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x63 pc=0x7ffbe20841d9]

runtime stack:
runtime.throw(0xf62a20, 0x2a)
	/usr/local/go/src/runtime/panic.go:547 +0x90
runtime.sigpanic()
	/usr/local/go/src/runtime/sigpanic_unix.go:12 +0x5a

goroutine 404 [sHigh five for submitting a report!
To avoid additional back and forth, please include the following details as relevant:
yscall, locked to thread]:
runtime.cgocall(0xafdea0, 0xc82049fbd8, 0x0)
	/usr/local/go/src/runtime/cgocall.go:123 +0x11b fp=0xc82049fb78 sp=0xc82049fb48
net._C2func_getaddrinfo(0x7ffbcc0008c0, 0x0, 0xc82047d320, 0xc8203fe198, 0x0, 0x0, 0x0)
	??:0 +0x55 fp=0xc82049fbd8 sp=0xc82049fb78
net.cgoLookupIPCNAME(0xc820010213, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/cgo_unix.go:111 +0x448 fp=0xc82049fd50 sp=0xc82049fbd8
net.cgoLookupIP(0xc820010213, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x409f45)
	/usr/local/go/src/net/cgo_unix.go:163 +0x56 fp=0xc82049fda8 sp=0xc82049fd50
net.lookupIP(0xc820010213, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup_unix.go:67 +0x94 fp=0xc82049fe18 sp=0xc82049fda8
net.glob.func16(0x101f828, 0xc820010213, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/hook.go:10 +0x4d fp=0xc82049fe58 sp=0xc82049fe18
net.lookupIPDeadline.func1(0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup.go:106 +0x71 fp=0xc82049fed8 sp=0xc82049fe58
internal/singleflight.(*Group).doCall(0x15ec1c0, 0xc82061e640, 0xc820010213, 0x7, 0xc8205530a0)
	/usr/local/go/src/internal/singleflight/singleflight.go:93 +0x2c fp=0xc82049ff88 sp=0xc82049fed8
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1 fp=0xc82049ff90 sp=0xc82049ff88
created by internal/singleflight.(*Group).DoChan
	/usr/local/go/src/internal/singleflight/singleflight.go:86 +0x3ee

goroutine 1 [select]:
net/http.(*Transport).getConn(0xc82045b740, 0xc8201b95e0, 0xc82001c680, 0xc820481200, 0x5, 0xc820552fa0, 0x18, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:711 +0x4ef
net/http.(*Transport).RoundTrip(0xc82045b740, 0xc8201b95e0, 0xc82045b740, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:311 +0x7e9
net/http.send(0xc8201b95e0, 0x7ffbe44e35c0, 0xc82045b740, 0x0, 0x0, 0x0, 0xc8204818c0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:260 +0x6b7
net/http.(*Client).send(0xc82047cc00, 0xc8201b95e0, 0x0, 0x0, 0x0, 0xc800000002, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:155 +0x185
net/http.(*Client).doFollowingRedirects(0xc82047cc00, 0xc8201b95e0, 0x101fa50, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:475 +0x8a4
net/http.(*Client).Do(0xc82047cc00, 0xc8201b95e0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:188 +0xff
github.com/cloudfoundry/cli/cf/net.Gateway.doRequest(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:432 +0x144
github.com/cloudfoundry/cli/cf/net.Gateway.doRequestAndHandlerError(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:404 +0x5e
github.com/cloudfoundry/cli/cf/net.Gateway.doRequestHandlingAuth(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:375 +0x190
github.com/cloudfoundry/cli/cf/net.Gateway.performRequestForResponseBytes(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:254 +0x8d
github.com/cloudfoundry/cli/cf/net.Gateway.PerformRequestForJSONResponse(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:274 +0x9a
github.com/cloudfoundry/cli/cf/net.Gateway.GetResource(0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ffbe44ea690, 0xc82038c480, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:102 +0x160
github.com/cloudfoundry/cli/cf/api/authentication.UAAAuthenticationRepository.GetLoginPromptsAndSaveUAAServerURL(0x7ffbe44ea798, 0xc82038c480, 0x0, 0x0, 0x101ea50, 0x0, 0x12a05f200, 0x0, 0x0, 0x0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/api/authentication/authentication.go:172 +0x1fb
github.com/cloudfoundry/cli/cf/api/authentication.(*UAAAuthenticationRepository).GetLoginPromptsAndSaveUAAServerURL(0xc8200d0000, 0xe58418, 0x0, 0x0)
	<autogenerated>:5 +0xbd
github.com/cloudfoundry/cli/cf/commands.Login.authenticate(0x7ffbe44ea4f8, 0xc8203450c0, 0x7ffbe44ea798, 0xc82038c480, 0x7ffbe44eaa80, 0xc8200d0000, 0x7ffbe44eace0, 0xc82001f680, 0x7ffbe44eae20, 0xc82001ac80, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/commands/login.go:169 +0xe4
github.com/cloudfoundry/cli/cf/commands.(*Login).Execute(0xc82040a2a0, 0x7ffbe44ebfc0, 0xc82000a440)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/commands/login.go:108 +0x3e7
main.main()
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/main/main.go:118 +0x1782

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 5 [syscall]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x132
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 426 [IO wait]:
net.runtime_pollWait(0x7ffbe22c1bc0, 0x72, 0xc82044f000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8200d6220, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8200d6220, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8200d61c0, 0xc82044f000, 0x1000, 0x1000, 0x0, 0x7ffbe44df050, 0xc82000e210)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc820312bb0, 0xc82044f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
crypto/tls.(*block).readFromUntil(0xc820482600, 0x7ffbe22c1d48, 0xc820312bb0, 0x5, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:460 +0xcc
crypto/tls.(*Conn).readRecord(0xc82031c000, 0x101ff17, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:562 +0x2d1
crypto/tls.(*Conn).Read(0xc82031c000, 0xc82044e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:939 +0x167
net/http.noteEOFReader.Read(0x7ffbe24a3ab0, 0xc82031c000, 0xc8200643a8, 0xc82044e000, 0x1000, 0x1000, 0x40a183, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:1687 +0x67
net/http.(*noteEOFReader).Read(0xc82055a000, 0xc82044e000, 0x1000, 0x1000, 0xc82028fd0d, 0x0, 0x0)
	<autogenerated>:284 +0xd0
bufio.(*Reader).fill(0xc82039e060)
	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
bufio.(*Reader).Peek(0xc82039e060, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/bufio/bufio.go:132 +0xcc
net/http.(*persistConn).readLoop(0xc820064340)
	/usr/local/go/src/net/http/transport.go:1073 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:857 +0x10a6

goroutine 427 [select]:
net/http.(*persistConn).writeLoop(0xc820064340)
	/usr/local/go/src/net/http/transport.go:1277 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:858 +0x10cb

goroutine 403 [select]:
net.lookupIPDeadline(0xc820010213, 0x7, 0xeceda44be, 0x2336afd6, 0x15ed2a0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup.go:109 +0x6a6
net.internetAddrList(0xe59db0, 0x3, 0xc820010213, 0xa, 0xeceda44be, 0x2336afd6, 0x15ed2a0, 0x0, 0x0, 0x0, ...)
	/usr/local/go/src/net/ipsock.go:252 +0x6ee
net.resolveAddrList(0xe55650, 0x4, 0xe59db0, 0x3, 0xc820010213, 0xa, 0xeceda44be, 0x2336afd6, 0x15ed2a0, 0x0, ...)
	/usr/local/go/src/net/dial.go:158 +0x466
net.(*Dialer).Dial(0xc82061e280, 0xe59db0, 0x3, 0xc820010213, 0xa, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/dial.go:216 +0x124
net.(*Dialer).Dial-fm(0xe59db0, 0x3, 0xc820010213, 0xa, 0x0, 0x0, 0x0, 0x0)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:458 +0x6e
net/http.(*Transport).dial(0xc82045b740, 0xe59db0, 0x3, 0xc820010213, 0xa, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:659 +0x96
net/http.(*Transport).dialConn(0xc82045b740, 0xc82001c680, 0xc820481200, 0x5, 0xc820552fa0, 0x18, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:762 +0x1e3d
net/http.(*Transport).getConn.func4(0xc82045b740, 0xc82001c680, 0xc820481200, 0x5, 0xc820552fa0, 0x18, 0xc820359e00)
	/usr/local/go/src/net/http/transport.go:706 +0x66
created by net/http.(*Transport).getConn
	/usr/local/go/src/net/http/transport.go:708 +0x262

### Platform Details

Fefora 23

### Shell

gnome-terminal

**Comment by dkoper** [comment]
We'll take a look!

**Comment by XenoPhex** [comment]
@eaceto What is the result if you first run `export GODEBUG=netdns=go` and then run the same commands which failed?

**Comment by amenzhinsky** [comment]
The same is for the arch linux `4.5.4-1-ARCH`, fixed by building it by myself

```
$ cf login 

API endpoint> api.local.pcfdev.io
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x0 pc=0x0]

runtime stack:
runtime.throw(0xf62a20, 0x2a)
	/usr/local/go/src/runtime/panic.go:547 +0x90
runtime.sigpanic()
	/usr/local/go/src/runtime/sigpanic_unix.go:12 +0x5a

goroutine 817 [syscall, locked to thread]:
runtime.cgocall(0xafdea0, 0xc82003dbd8, 0x0)
	/usr/local/go/src/runtime/cgocall.go:123 +0x11b fp=0xc82003db78 sp=0xc82003db48
net._C2func_getaddrinfo(0x7ff6980008c0, 0x0, 0xc820126090, 0xc82002e220, 0x0, 0x0, 0x0)
	??:0 +0x55 fp=0xc82003dbd8 sp=0xc82003db78
net.cgoLookupIPCNAME(0xc8201c3380, 0x13, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/cgo_unix.go:111 +0x448 fp=0xc82003dd50 sp=0xc82003dbd8
net.cgoLookupIP(0xc8201c3380, 0x13, 0x0, 0x0, 0x0, 0x0, 0x0, 0x409f45)
	/usr/local/go/src/net/cgo_unix.go:163 +0x56 fp=0xc82003dda8 sp=0xc82003dd50
net.lookupIP(0xc8201c3380, 0x13, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup_unix.go:67 +0x94 fp=0xc82003de18 sp=0xc82003dda8
net.glob.func16(0x101f828, 0xc8201c3380, 0x13, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/hook.go:10 +0x4d fp=0xc82003de58 sp=0xc82003de18
net.lookupIPDeadline.func1(0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup.go:106 +0x71 fp=0xc82003ded8 sp=0xc82003de58
internal/singleflight.(*Group).doCall(0x15ec1c0, 0xc820468a00, 0xc8201c3380, 0x13, 0xc8201c3480)
	/usr/local/go/src/internal/singleflight/singleflight.go:93 +0x2c fp=0xc82003df88 sp=0xc82003ded8
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1 fp=0xc82003df90 sp=0xc82003df88
created by internal/singleflight.(*Group).DoChan
	/usr/local/go/src/internal/singleflight/singleflight.go:86 +0x3ee

goroutine 1 [select]:
net/http.(*Transport).getConn(0xc8202223c0, 0xc82046e000, 0x0, 0xc8201d7e90, 0x5, 0xc8201c3380, 0x17, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:711 +0x4ef
net/http.(*Transport).RoundTrip(0xc8202223c0, 0xc82046e000, 0xc8202223c0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:311 +0x7e9
net/http.send(0xc82046e000, 0x7ff6b6385680, 0xc8202223c0, 0x0, 0x0, 0x0, 0xc8201a49c0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:260 +0x6b7
net/http.(*Client).send(0xc8201d7ef0, 0xc82046e000, 0x0, 0x0, 0x0, 0xc800000000, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:155 +0x185
net/http.(*Client).doFollowingRedirects(0xc8201d7ef0, 0xc82046e000, 0x101fa50, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:475 +0x8a4
net/http.(*Client).Do(0xc8201d7ef0, 0xc82046e000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:188 +0xff
github.com/cloudfoundry/cli/cf/net.Gateway.doRequest(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:432 +0x144
github.com/cloudfoundry/cli/cf/net.Gateway.doRequestAndHandlerError(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:404 +0x5e
github.com/cloudfoundry/cli/cf/net.Gateway.doRequestHandlingAuth(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:375 +0x190
github.com/cloudfoundry/cli/cf/net.Gateway.performRequestForResponseBytes(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:254 +0x8d
github.com/cloudfoundry/cli/cf/net.Gateway.PerformRequestForJSONResponse(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:274 +0x9a
github.com/cloudfoundry/cli/cf/net.Gateway.GetResource(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:102 +0x160
github.com/cloudfoundry/cli/cf/api.RemoteInfoRepository.getCCAPIInfo(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/api/endpoints.go:51 +0x12c
github.com/cloudfoundry/cli/cf/api.RemoteInfoRepository.GetCCInfo(0x7ff6b7bca058, 0xc82012abb0, 0x101ea38, 0x1, 0x12a05f200, 0x0, 0x0, 0x0, 0x7ff6b7bc9cb0, 0xc8201949c0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/api/endpoints.go:33 +0x20e
github.com/cloudfoundry/cli/cf/api.(*RemoteInfoRepository).GetCCInfo(0xc820390480, 0xc8201c2700, 0x13, 0x40000000, 0x0, 0x0, 0x0, 0x0)
	<autogenerated>:31 +0xf1
github.com/cloudfoundry/cli/cf/configuration/coreconfig.APIConfigRefresher.Refresh(0x7ff6b7bca268, 0xc820390480, 0x7ff6b7bc9db8, 0xc8201949c0, 0xc8201c2700, 0x13, 0x0, 0x0, 0x0, 0x0)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/configuration/coreconfig/api_config_refresher.go:24 +0x88
github.com/cloudfoundry/cli/cf/commands.Api.setAPIEndpoint(0x7ff6b7bc9b18, 0xc8201492c0, 0x7ff6b7bca268, 0xc820390480, 0x7ff6b7bc9db8, 0xc8201949c0, 0xc8201c2700, 0x13, 0xc82029a201, 0xe573b0, ...)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/commands/api.go:97 +0x307
github.com/cloudfoundry/cli/cf/commands.(*Login).Execute(0xc820414480, 0x7ff6b7bcb620, 0xc82019ea80)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/commands/login.go:88 +0x1f8
main.main()
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/main/main.go:118 +0x1782

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 20 [syscall]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x132
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 815 [select]:
net.lookupIPDeadline(0xc8201c3380, 0x13, 0xecee20c00, 0x299a2a14, 0x15ed2a0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/lookup.go:109 +0x6a6
net.internetAddrList(0xe59db0, 0x3, 0xc8201c3380, 0x17, 0xecee20c00, 0x299a2a14, 0x15ed2a0, 0x0, 0x0, 0x0, ...)
	/usr/local/go/src/net/ipsock.go:252 +0x6ee
net.resolveAddrList(0xe55650, 0x4, 0xe59db0, 0x3, 0xc8201c3380, 0x17, 0xecee20c00, 0x299a2a14, 0x15ed2a0, 0x0, ...)
	/usr/local/go/src/net/dial.go:158 +0x466
net.(*Dialer).Dial(0xc8204685f0, 0xe59db0, 0x3, 0xc8201c3380, 0x17, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/dial.go:216 +0x124
net.(*Dialer).Dial-fm(0xe59db0, 0x3, 0xc8201c3380, 0x17, 0x0, 0x0, 0x0, 0x0)
	/tmp/build/80754af9/gopath/src/github.com/cloudfoundry/cli/cf/net/gateway.go:458 +0x6e
net/http.(*Transport).dial(0xc8202223c0, 0xe59db0, 0x3, 0xc8201c3380, 0x17, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:659 +0x96
net/http.(*Transport).dialConn(0xc8202223c0, 0x0, 0xc8201d7e90, 0x5, 0xc8201c3380, 0x17, 0xc820421c80, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:762 +0x1e3d
net/http.(*Transport).getConn.func4(0xc8202223c0, 0x0, 0xc8201d7e90, 0x5, 0xc8201c3380, 0x17, 0xc8200162a0)
	/usr/local/go/src/net/http/transport.go:706 +0x66
created by net/http.(*Transport).getConn
	/usr/local/go/src/net/http/transport.go:708 +0x262
```

**Comment by challiwill** [comment]
@amenzhinsky can you please try using this new binary that we have generated and see if it fixes the issue for you.

We uploaded it to https://s3.amazonaws.com/temp-ci-binary-ubuntu/cf-netgo

**Comment by amenzhinsky** [comment]
@challiwill yes it works, now I can login

**Comment by XenoPhex** [comment]
@amenzhinsky Thanks for the quick response! We believe that this is the same issue as in #843 which appeared to affect another user's arch system as well. So hopefully we have properly fixed this issue for everyone.

**Comment by ssprabhu** [comment]
Looks like the problem is to do with the disabling ssl3 on Fedora 23.

http://devel.fedoraproject.narkive.com/qDWcA5DS/f23-system-wide-change-disable-ssl3-and-rc4-by-default

**Comment by dkoper** [comment]
Thanks for sharing that.

We have fixed this panic in cf CLI 6.19.0, released today.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM
